admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,193 @@
+==========================
+Configuring Memory Caching
+==========================
+
+You can significantly improve your ownCloud server performance with memory 
+caching, where frequently-requested objects are stored in memory for faster 
+retrieval. There are two types of caches to use: PHP opcache, and data caching 
+for your Web server. If you do not install a memcache you will see a warning on 
+your ownCloud admin page. (A memcache is not required and you may ignore the warning if you prefer.)
+
+A PHP opcache stores compiled PHP scripts so they don't need to be re-compiled 
+every time they are called. PHP bundles an opcache since version 5.5, so you 
+don't need to install or configure it.
+
+If you are using PHP 5.4, which is the oldest supported PHP version for 
+ownCloud, you may install the Alternative PHP Cache (APC). This is both an 
+opcache and data cache. APC has not been updated since 2012 and is essentially 
+dead, and PHP 5.4 is very old and lags far behind later releases. If it is possible 
+to upgrade to a later PHP release that is the best option.
+
+Data caching is supplied by the Alternative PHP Cache, user (APCu) in PHP 
+5.5+, Memcached, or Redis.
+
+ownCloud supports multiple memory caching backends, so you can choose the type 
+of memcache that best fits your needs. The supported caching backends are:
+
+* `APC <http://php.net/manual/en/book.apc.php>`_ 
+   A local cache for systems running PHP 5.4.
+* `APCu <https://pecl.php.net/package/APCu>`_ 
+   A local cache for systems running PHP 5.5 and up, APCu 4.06 or better required.
+* `Memcached <http://www.memcached.org/>`_ 
+   Distributed cache for multi-server ownCloud installations.
+* `Redis <http://redis.io/>`_ 
```

Comment:
Needs php5-redis in version 2.2.5 or higher: https://github.com/owncloud/core/blob/master/lib/private/memcache/redis.php#L197

Note: This is the version of the php module, not redis itself.